package.json

```diff
@@ -67,5 +68,6 @@
         "jquery:$"
       ]
     }
-  }
+  },
+  "dependencies": {}
```

Comment:
nope sorry, i used it commandline to uninstall and reinstall as --save-dev and didn't realized I'd left that in.